CHANGELOG.rst

```diff
@@ -17,6 +17,8 @@ History
 - Unify tracer fixture and move it to conftest.py
 - Fix some deprecation warnings
 - Clean up imports
+- Add ability to set ``response_handler_hook`` using ``set_response_handler_hook`` method of ``RequestsPatcher``
```

Comment:
could we add this to the README somewhere?

Reply:
I've added the `Customization` section to the bottom of the `Usage` section of the README.